From 9b9c4d2f8412db2541de8a3a8ef4aeb6896300df Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Wed, 31 Jan 2018 09:55:36 +0100
Subject: [PATCH 2/2] overlay: Allow to use UnifiedNlp location provider

This doesn't have security implications as in order to use the location
provider in needs to be installed as a system app. However this would
allow it to be flashed to system like gapps.

Change-Id: I120b8dc85dfbbe0b4d9e0a620c58d7556feca9c5
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 4f58448..f5673a8 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -17,6 +17,8 @@
 -->
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string-array name="config_locationProviderPackageNames" translatable="false">
+        <!-- The UnifiedNlp provider by MicroG -->
+        <item>org.microg.nlp</item>
         <!-- The Google provider -->
         <item>com.google.android.gms</item>
         <!-- The standard AOSP fused location provider -->
-- 
2.29.2

